lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -59,8 +59,24 @@
                 });
             },
 
-            visitBookmark: function (event) {
-                window.location = event.target.pathname;
+            visitBookmark: function (url) {
+                window.location.href = url;
+            },
+
+            logBookmarkedItem: function (event) {
+                var self = this;
+                var bookmark_id = event.currentTarget.dataset.id;
+                var component_usage_id = bookmark_id.split(',')[1];
```

Comment:
I would put the `usage_id` as a data attribute on the element and picking it here rather than doing parsing. Ids should be treated as opaque identifiers and not having any meaning.